### PR TITLE
D2IQ-88102: add `allowMultipleInstaces` key to catalog applications

### PR DESCRIPTION
**Pull request by mstruebing**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-88102

## Description of changes being made

I've added the `allowMultipleInstaces` key to the catalog app definition tables.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4454.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by mstruebing** [comment]
Can we ignore the value error? I only changed the formatting on the line which throws an error.
If not: How should I change the wording?

File: pages/dkp/kommander/2.3/projects/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 1]
```diff
@@ -10,14 +10,15 @@ You can define how custom applications display in the DKP UI by defining a `meta
...
+| Field                  | Default              | Description                                                                                     |
+| ---------------------- | -------------------- | ----------------------------------------------------------------------------------------------- |
+| displayName            | falls back to App ID | Display name of the application for the UI.                                                     |
+| description            | ""                   | Short description, should be a sentence or two, displayed in the UI on the application card.    |
+| category               | general              | 1 or more categories for this application. Categories are used to group applications in the UI. |
+| overview               |                      | Markdown overview used on the application detail page in the UI.                                |
+| icon                   |                      | Base64 encoded icon SVG file used for application logos in the UI.                              |
+| scope                  | workspace            | List of scopes, can be workspace and/or project currently.                                      |
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.3/projects/applications/catalog-applications/custom-applications/application-metadata/index.md:20]: We could say: 

> List of scopes, currently can be workspace, project, or both.

**Review by gvasquezvargas — APPROVED** [review]
Other than the comment I left to resolve Vale's error, LGTM!

**Review by nataliepina — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.3/projects/applications/catalog-applications/custom-applications/application-metadata/index.md (hunk 1 above)
Comment (mstruebing) [reply pages/dkp/kommander/2.3/projects/applications/catalog-applications/custom-applications/application-metadata/index.md]: I've changed it, thank you for that suggestion 🙌

**Pull request merged**